docs/source/integrations/pgvector.rst

```diff
@@ -0,0 +1,639 @@
+.. pgvector-integration:
+
+PostgreSQL Vector Search Integration
+=================================
```

Comment:
nit: Sphinx emits errors if the `===` don't exactly match the length of the text above them